synapse/config/registration.py

```diff
@@ -84,11 +84,11 @@ def default_config(self, generate_secrets=False, **kwargs):
         #
         # allowed_local_3pids:
         #     - medium: email
-        #       pattern: ".*@matrix\\.org"
+        #       pattern: '.*@matrix\\.org'
```

Comment:
I'd have thought this would now have one too many escapes?

Reply:
Right, but single slashes is what you want in the yaml if its going to be fed to regex?

Reply:
OH sorry, I'm with you